core/domain/improvements_services_test.py

```diff
@@ -27,28 +27,40 @@
 from core.platform import models
 from core.tests import test_utils
 
+from typing_extensions import Final
+
+MYPY = False
+if MYPY: # pragma: no cover
+    from mypy_imports import improvements_models
+
 (improvements_models,) = (
     models.Registry.import_models([models.NAMES.improvements]))
 
 
 class ImprovementsServicesTestBase(test_utils.GenericTestBase):
     """Base class with helper methods for the improvements_services tests."""
 
-    EXP_ID = 'eid'
-    MOCK_DATE = datetime.datetime(2020, 6, 15)
+    EXP_ID: Final = 'eid'
+    MOCK_DATE: Final = datetime.datetime(2020, 6, 15)
 
-    def setUp(self):
+    def setUp(self) -> None:
         super(ImprovementsServicesTestBase, self).setUp()
         self.signup(self.OWNER_EMAIL, self.OWNER_USERNAME)
-        self.owner_id = self.get_user_id_from_email(self.OWNER_EMAIL)
+        self.owner_id = self.get_user_id_from_email(self.OWNER_EMAIL)  # type: ignore[no-untyped-call]
         self.exp = self.save_new_valid_exploration(self.EXP_ID, self.owner_id)
         # Necessary to provide sufficient debug information when failures occur.
-        self.maxDiff = None
+
+        # Internally variable 'maxDiff' is defined as a strict integer variable
+        # but here we are assigning None value to it, which cause MyPy to throw
+        # an assignment error. Thus to avoid an error, we used ignore here.
+        self.maxDiff = None  # type: ignore[assignment]
```

Comment:
Can we assign some equal numeric value?

Reply:
I found that this `maxdiff` value is never used in the test, so I removed it.

Reply:
I assigned maxdiff with `0`.